File: backend/BusinessService/Common/Enums/CmEnums.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: システム共通(業務)
//             区分列挙型
// 作成日　　: 2022.12.12
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.Service
{
    public enum EnumRuntimeMode
    {
        NORMAL,
        DEBUG,
        STUB
    }

    public enum EnumDownloadType
    {
        File,
        Data
    }
}

File: backend/BusinessService/Common/Enums/CmEnums.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: システム共通(業務)
'             区分列挙型
' 作成日　　: 2024.07.12
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Service
    Public Enum EnumRuntimeMode
        NORMAL
        DEBUG
        STUB
    End Enum

    Public Enum EnumDownloadType
        File
        Data
    End Enum
End Namespace
